containers/ReportPage.js

```diff
@@ -344,6 +349,20 @@ class ReportPage extends React.Component {
     return name;
   }
 
+    getCode(searchWithCode){
```

Comment:
There are at least two here that should be taken care of. 
1. Code (.o) not provided,
2. Structure is not the same (let's try to uniform this instead)

The word 'code' is a bit ambigious, too. Maybe use OrgCode?

`  getOrgCode(searchWithCode) {
    if (!searchWithCode || !this.props.code) return null;

    const codeJSON = JSON.parse(this.props.code);
    const orgCode = codeJSON.o ? codeJSON.o.toLowerCase() : null;

    if (codeJSON !== window.config.netexPrefix.toLowerCase()) {
      return orgCode;
    }
    return null;
  }`